apis/r/R/SOMATileDBContext.R

```diff
@@ -0,0 +1,145 @@
+#' SOMA TileDB Context
+#'
+#' Context map for TileDB-backed SOMA objects
+#'
+#' @export
+#'
+#' @noMd
+#'
+SOMATileDBContext <- R6::R6Class(
+  classname = 'SOMATileDBContext',
+  inherit = SOMAContextBase,
+  public = list(
+    #' @template param-config
+    #' @param cached Force new creation
+    #'
+    #' @return An instantiated \code{SOMATileDBContext} object
+    #'
+    initialize = function(config = NULL, cached = TRUE) {
+      # super$initialize()
+      config <- config %||% character()
+      # Identify options that are SOMA-specific
+      soma_opts <- which(x = names(x = config) %in% names(x = .SOMA_CONTEXTS()))
+      if (length(x = soma_opts)) {
+        soma_config <- config[soma_opts]
+        config <- config[-soma_opts]
+      } else {
+        soma_config <- NULL
+      }
+      super$initialize(config = soma_config)
+      if (is.list(x = config)) {
+        config <- unlist(x = config)
+      }
+      stopifnot(
+        !length(x = config) || is.character(x = config),
+        !length(x = config) || is_named(x = config, allow_empty = FALSE)
+      )
+      config['sm.mem.reader.sparse_global_order.ratio_array_data'] <- '0.3'
+      # Add the TileDB context
+      # TODO: Reenable this with newer version of tiledb-r
+      # if (!length(x = config)) {
+      #   config <- NA_character_
+      # }
+      # cfg <- tiledb::tiledb_config(config = config)
+      # TODO: replace this when patched version of tiledb-r is released
```

Comment:
(May need a versioned Imports: then just to be safe the bug-fixed version is used.)